packages/package_info_plus/package_info_plus_web/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_info_plus_web
 description: Web platform implementation of package_info_plus
-version: 1.0.5
+version: 1.0.6
```

Comment:
i just realized, since this is a new feature, would it make sense to increase the minor version instead? `1.1.0`? not sure how this should work

Reply:
also, should have updated the version in `package_info_plus` package. will raise a separate PR for that after resolving my previous doubt.

Reply:
No worries 😊

@miquelbeltran possibly one more federated version fix for you ✌🏽